### PR TITLE
French Localization.

### PR DESCRIPTION
**Pull request by el00ruobuob**
Rebased & squash to a single commit for @luigi1111 & @erciccione not to repead themself one more time ;-)

It does include:
- French Localization creation
- Main pages translations
- Get-started pages translations
- community pages translations
- Main ressources pages translations
- User-guides translations

It does not includes yet:
- Moneropedia pages translations
- Developer-guides pages translations.

Disclaimer: Several technical terms are translated, like "blockchain" or "mnemonic seeds" for instance. I'm not a fan of those but French language loves to translate everything, even if the result looks ugly to tech guys like us. But i did this accordingly to the actual Monero-GUI french translation (to be consistent, at least).

@erciccione it's up to you to try. If you remind me how to do it, i might be able to build the site locally and check by myself.
Let me know.

I'll do the Moneropedia and developer-guides soon.

### CODE REVIEW

File: _i18n/fr.yml [hunk 1]
```diff
@@ -0,0 +1,97 @@
+langs:
+  en: English
+  es: Español
+  fr: Français
```
Comment (erciccione) [inline _i18n/fr.yml]: Please add Italian before `fr`. It should look like this:
```
langs:
en: English
es: Español
it: Italian
fr: Français
```

**Comment by el00ruobuob** [comment]
@erciccione done. do you need me to rebase on fluffy's latest merge?

**Comment by erciccione** [comment]
about building the website, you can find instructions [in the readme](https://github.com/monero-project/monero-site#20-what-youll-need)

**Comment by erciccione** [comment]
Please also add "French" in `en.yml` `es.yml` and `it.yml`.

**Comment by el00ruobuob** [comment]
And template.yml? did it anyway

File: _i18n/fr/index.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+                </div>
+                <!-- end center one-third block-->
+
+                <!-- right one-third block-->
+                <div class="right one-third col-lg-4 col-md-4 col-sm-12 col-xs-12">
+                    <div class="info-block">
+                        <div class="row center-xs">
+                            <div class="col">
+                                <h2>FAQ</h2>
+                                <p class="main-resources">Nous avons entendu beaucoup de questions au cours des années et avons compilé, pour votre commodité, une FAQ complète et variée. Ne vous inquiétez pas, si vos questions ne sont pas ici, vous pouvez toujours demander à la communauté.</p>
```
Comment (erciccione) [inline _i18n/fr/index.md]: This text is too long. It ends outside the box

**Comment by erciccione** [comment]
@glv2 @cryptochangements34  @assylias @Neozaru

Hey folks, this is the French translation of the website, @el00ruobuob has done a lot of work and the localization is basically done, Could you review it? Thanks!

File: _i18n/fr/index.md (hunk 2 above)
Comment (el00ruobuob) [reply _i18n/fr/index.md]: Rephrased to 196 characters.

File: _data/lang/fr/navigation.yml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    url: get-started/faq
+- title: Téléchargements
+  url: downloads/
+- title: Actualités
+  subfolderitems:
+    - page: Toutes
+      url: blog
+    - page: Missives
+      url: blog/tags/monero%20missives.html
+    - page: Journaux de Réunions
```
[the remark this answers is not in the document]
Comment (el00ruobuob) [reply _data/lang/fr/navigation.yml]: ok

File: _i18n/fr/resources/user-guides/solo_mine_GUI.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+Il est très facile de miner en solo avec la GUI officielle. Si vous ne l'avez pas déjà fait, allez sur la <a href="{{site.baseurl}}/downloads/">pade de téléchargement Monero</a> et téléchargez la GUI officielle pour votre système d'exploitation. Puis, lancez l'installation et patientez pendant que Monero se synchronise avec le réseau. Vous devriez voir qu'elle insique "Connecté" dans le coin en bas à gauche.
```
[the remark this answers is not in the document]
Comment (el00ruobuob) [reply _i18n/fr/resources/user-guides/solo_mine_GUI.md]: ok

File: _i18n/fr/resources/user-guides/solo_mine_GUI.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+Cliquez sur l'onglet "Anvancé". Vous devriez voir aparaitre d'autres options. Cliquez maintenant sur le sous-onglet "Mine".
+
+<img src="/resources/user-guides/png/solo_mine_GUI/02.PNG" style="width: 600px;"/>
+
+Vous devriez maintenant avoir une option pour démarrer l'extraction minière. Vous pouvez au besoin changer le nombre de threads avec lesquels miner. Pour plus d'efficacité, vous devriez miner avec le cache de votre CPU en Mo divisé par 2. Vous aurez besoin de vérifier les caractéristiques de votre CPU sur le site du constructeur. Si vous n'êtes pas sûr, laissez le nombre de threads à 1. Cliquez sur Démarrer.
+
+<img src="/resources/user-guides/png/solo_mine_GUI/03.PNG" style="width: 600px;"/>
+
+Vous miner maintenant avec le réseau, comme vous pouvez le voir en bas de l'image. Dans cet exemple, l'ordinateur contribue à hauteur de 23 H/s au réseau Monero. l'extraction minière aide à maintenir le réseau sécurisé, et vous pourriez avoir de la chance et recevoir une récompense pour avoir protégé le réseau.
```
[the remark this answers is not in the document]
Comment (el00ruobuob) [reply _i18n/fr/resources/user-guides/solo_mine_GUI.md]: ok

File: _i18n/fr/resources/user-guides/view_only.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
+Un portefeuille d'audit ne peut que voir quelles transactions entrantes vous appartiennent. Il ne peut pas dépenser vos Moneroj, en fait il ne peut même pas voir les transactions sortante de votre portefeuille. Cela rends le portefeuille d'audit particulièrement intéressant pour :
```
[the remark this answers is not in the document]
Comment (el00ruobuob) [reply _i18n/fr/resources/user-guides/view_only.md]: ok

File: _i18n/fr/resources/user-guides/view_only.md [hunk 7]
```diff
@@ -0,0 +1,12 @@
+Un portefeuille d'audit ne peut que voir quelles transactions entrantes vous appartiennent. Il ne peut pas dépenser vos Moneroj, en fait il ne peut même pas voir les transactions sortante de votre portefeuille. Cela rends le portefeuille d'audit particulièrement intéressant pour : 
+
+* Les déceloppeurs écrivant des librairies de validation de paiements ;
+* Les utilisateurs validant les transactions entrantes sur des portefeuilles froids.
+
+### Créer un portefeuille d'audit
+
+Vous pouvez créer un portefeuille d'audit depuis n'importe quel portefeuille existant.
+
+Ouvrez un portefeuille existant ou créez en un nouveau en utilisant `monero-wallet-cli`. Dans le portefeuille, tapez `address` et `viewkey` pour afficher l'adresse et la clefs privée d'audit du portefeuille. Tapez `exit` pour fermer le portefeuille.
```
[the remark this answers is not in the document]
Comment (el00ruobuob) [reply _i18n/fr/resources/user-guides/view_only.md]: ok

File: _i18n/fr/resources/user-guides/view_only.md [hunk 8]
```diff
@@ -0,0 +1,12 @@
...
+
+Ensuite, créez votre portefeuille d'audit en tapant `monero-wallet-cli --generate-from-view-key nom-du-portefeuille`. Le dernier argument sera le nom du fichier de votre portefeuille. Vos `adresse standard` (`Standard address`) et `clef d'audit` (`View key`) vous seront demandées par le portefeuille. Collé l'adresse originelle de votre portefeuille et la clef privée d'audit. Puis, tapez Entrer et confirmez un mot de passe pour pour votre nouveau portefeuille afin de terminer l'opération.
```
[the remark this answers is not in the document]
Comment (el00ruobuob) [reply _i18n/fr/resources/user-guides/view_only.md]: collez : ok
La touche Entrer

File: _i18n/fr/resources/user-guides/vps_run_node.md [hunk 9]
```diff
@@ -0,0 +1,47 @@
+# monerod
+
+`monerod` est le démon est un logiciel qui est inclus dans la suite Monero. C'est un programme en ligne de commande qui gère la chaîne de blocs. Tandis que le portefeuille Bitcoin gère à la fois un compte et la chaîne de blocs, Monero sépare ces composants : `monerod` gère la chaîne de blocs, et `monero-wallet-cli` gère le compte.
```
[the remark this answers is not in the document]
Comment (el00ruobuob) [reply _i18n/fr/resources/user-guides/vps_run_node.md]: ok

File: _i18n/fr/resources/user-guides/vps_run_node.md [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+
+Ce guide supposes que vous avez déjà créé un compte VPS et que vous utilisez SSH pour accéder à la console du serveur.
```
[the remark this answers is not in the document]
Comment (el00ruobuob) [reply _i18n/fr/resources/user-guides/vps_run_node.md]: ok

File: _i18n/fr/the-monero-project.md [hunk 11]
```diff
@@ -0,0 +1,44 @@
...
+        <div class="left half no-pad-sm col-lg-6 col-md-6 col-sm-12 col-xs-12">
+            <div class="info-block">
+                <div class="row center-xs">
+                    <div class="col">
+                        <h2>Kovri</h2>
+                    </div>
+                </div>
+<div class="row start-xs monero-project" markdown="1">
+
+Le projet Kovri utilise un chiffrement de bout-en-bout, si bien que ni l'émetteur ni le destinataire d'une transaction Monero n'ont à révéler leur adresse IP à l'autre côté, ni aux observateurs tiers (la chaîne de blocs). Ceci est réalisé en utilisant la même technologie qui fait fonctionner le dark net, i2p (Invisible Internet Protocol). Ce projet est actuellement en cours de dévelopement actif et intensif, et n'est pour le moment pas encore intégré à Monero.
```
[the remark this answers is not in the document]
Comment (el00ruobuob) [reply _i18n/fr/the-monero-project.md]: ok

File: _i18n/fr/the-monero-project.md [hunk 12]
```diff
@@ -0,0 +1,44 @@
...
+        <div class="right half col-lg-6 col-md-6 col-sm-12 col-xs-12">
+            <div class="info-block">
+                <div class="row center-xs">
+                    <div class="col">
+                        <h2>Open Alias</h2>
+                    </div>
+                </div>
+<div class="row start-xs monero-project" markdown="1">
+
+Le projet OpenAlias simplifie les paiements en cryptomonnaie en fournissant des FQDNs (Fully Qualified Domain Names, p.e. example.openalias.org) pour les adresses de portefeuilles Monero d'une manière qui garantit que la Confidentialité de chacun est sécurisée. Ce projet est bien avancé et a déjà été implémenté dans de nombreux portefeuilles Monero.
```
[the remark this answers is not in the document]
Comment (el00ruobuob) [reply _i18n/fr/the-monero-project.md]: ok

File: _i18n/fr/technical-specs.md [hunk 13]
```diff
@@ -0,0 +1,132 @@
...
+                        </div>
+                    </div>
+
+<div markdown="1">
+* infinie
+</div>
+
+                    <div class="row">
+                        <div class="col">
+                            <h3>Confidentialité de l'éméteur</h3>
```
[the remark this answers is not in the document]
Comment (el00ruobuob) [reply _i18n/fr/technical-specs.md]: ok

**Comment by el00ruobuob** [comment]
@assylias i hate myself for all those typos... Until you point them out, i didn't even see them by reading it again and again...
Regarding _i18n/fr/resources/user-guides/view_only.md line 12, as we talk about hitting Enter, should i change it to "tapez sur la touche Entrer", "appuyer sur la touche Entrer" or anything else?

**Comment by assylias** [comment]
No worries - you've done a great job!  I think it should be "Appuyer sur la touche Entrée" (note entrer).

**Comment by el00ruobuob** [comment]
so "appuyez sur la touche Entrée" it is.

**Comment by el00ruobuob** [comment]
I found some more "Entrer" that i corrected to "la touche Entrée"

**Comment by erciccione** [comment]
@el00ruobuob Just pull upstream so you have the correct base. You probably want to use `--rebase` when pulling

**Comment by el00ruobuob** [comment]
@mattcode55 I may be mis-explaining it... I do not refer to the pages already removed from the other languages by Rehar, but to the translated one that will not be changed by a rebase.

**Comment by mattcode55** [comment]
Oh, sorry. It's my fault for not reading properly ;)

The event page and link shouldn't be included in this PR, so you can
delete it. You can do it however you want (in a separate commit,
amending it to a previous one, ...), but I think the best way would be
to just make a commit for it now and squash it at the end.

**Comment by el00ruobuob** [comment]
@mattcode55 Thanks. It's clear now.

**Comment by el00ruobuob** [comment]
Rebased on latest, conflicts solved with polish translation.

**Comment by el00ruobuob** [comment]
@erciccione @glv2 @cryptochangements34 @assylias @Neozaru Anything else i should change here? I've often rebased to keep it up to date, and i'm available anytime for any update.

**Comment by erciccione** [comment]
@el00ruobuob You don't need to recommit all changes, just pull upstream and you'll be fine. I don't think this will create conflicts, but still...

**Comment by el00ruobuob** [comment]
@erciccione i had a few conflicts while rebasing after the polish translation merge, but it's fine now.
So i should not rebase this way? Just pull upstream?

**Pull request merged**